### PR TITLE
[ELY-2370] Move appropriate methods to the JaasRealmIdentity class from JaasSecurityRealm class

### PR DESCRIPTION
**Pull request by Kshitiz-Mhto**
Issue: [ELY-2370](https://issues.redhat.com/browse/ELY-2370)

### CODE REVIEW

File: auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java [hunk 1]
```diff
@@ -148,54 +148,10 @@ public SupportLevel getEvidenceVerifySupport(final Class<? extends Evidence> evi
...
-            return null;
-        } else {
-            return new JaasSecurityRealmDefaultCallbackHandler(principal, evidence);
-        }
-    }
-
     private class JaasRealmIdentity implements RealmIdentity {
 
         private final Principal principal;
+        private URI jaasConfigFilePath;
```
Comment (skyllarr) [inline auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: @Kshitiz-Mhto Thank you for the PR! Just a minor, this should not be added here, rather this class should use `jaasConfigFilePath` from its parent. You can try running `mvn clean install` locally to see if your changes pass.

Comment (skyllarr) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: @Kshitiz-Mhto Sorry I meant to say this class should use `jaasConfigFilePath` from its outer class, not parent

Comment (Kshitiz-Mhto) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: Hi @Skyllarr, when i removed the line 154 as per your suggestion, it leads to a error in line  

line264

which says `cannot resolve symbol `

---
then when i refactor the line 264 from `this.jaasConfigFilePath` ---> `jaasConfigFilePath` , error gone

Comment (skyllarr) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: @Kshitiz-Mhto Yes, changing `this.jaasConfigFilePath` to `jaasConfigFilepath` is a good fix! 

Feel free to update your PR with this change. If you encounter any problems with git, just let us know.

Comment (Kshitiz-Mhto) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: @Skyaller thank you for your warm welcome and for the opporunity. Another thing, i m having some issue with building this project in my intellij, should i post those error screenshot on  zulip?

Comment (skyllarr) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: @Kshitiz-Mhto Sure! Yes you can post those here or on Zulip, @fjuma or @cam-rod might be able to help you. 

Just some general tips for building, you should use Java 11 and have JAVA_HOME variable configured.

Comment (jessicarod7) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: Hey @Kshitiz-Mhto, we've just added some configuration guides for Elytron, [take a look](https://wildfly-security.github.io/wildfly-elytron/guides) and let me know if you have any issues.

Comment (Kshitiz-Mhto) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java]: Successfully  executed `sudo mvn clean install` thank you :+1:

**Comment by skyllarr** [comment]
Hello @Kshitiz-Mhto , currently there are 2 commits in this PR. Can you please squash the commits into a single commit with the ELY issue number at the beggining? Thank you!

**Comment by skyllarr** [comment]
@Kshitiz-Mhto We have a video that shows how to squash commits here: https://www.youtube.com/watch?v=NUfyS-2vTfo . If you need any help let us know. Thanks!

**Comment by Kshitiz-Mhto** [comment]
> Hello @Kshitiz-Mhto , currently there are 2 commits in this PR. Can you please squash the commits into a single commit with the ELY issue number at the beggining? Thank you!

hello! @Skyllarr  Done! thankyou

**Comment by skyllarr** [comment]
@Kshitiz-Mhto Looks great, thank you!

**Review by skyllarr — APPROVED** [review]

**Review by fjuma — APPROVED** [review]
Sorry for the delay in getting this merged, was away for a bit. Thanks @Kshitiz-Mhto!

**Pull request merged**